lib/wovnrb/headers.rb

```diff
@@ -141,7 +132,7 @@ def out(headers)
       if lang_code != @settings['default_lang'] && headers.key?('Location') && headers['Location'] =~ r && !@settings['ignore_globs'].ignore?(headers['Location'])
```

Comment:
(Also not related to the code)
Do we need to do anything for `custom_domain` here?
https://github.com/WOVNio/wovnrb/blob/6546e555a85737f8befd43a1fee4db9c69908163/lib/wovnrb/headers.rb#L141-L146

Reply:
Every switch statement on url_pattern must have something for custom domain

Reply:
Replied, and fixed in b54707187529a547040a3ea86482acdabe26dc63.

Reply:
Strange that tests seem to be failing in CI but is passing locally.